go.mod

```diff
@@ -142,8 +145,7 @@ replace github.com/opencontainers/image-spec => github.com/opencontainers/image-
 
 replace github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.3
 
-// TODO: remove when ibc-go upgrades cosmos-sdk version
-replace github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.44.6
+// needed due to a breaking change in 0.44.6
+replace github.com/cosmos/cosmos-sdk => github.com/crypto-org-chain/cosmos-sdk v0.44.8-patch
```

Comment:
https://github.com/cosmos/cosmos-sdk/compare/v0.44.6...crypto-org-chain:cosmos-sdk:v0.44.8-patch

Reply:
for cherry-picks https://github.com/cosmos/cosmos-sdk/compare/release/v0.44.x...crypto-org-chain:cosmos-sdk:v0.44.8-patch

Reply:
https://github.com/cosmos/cosmos-sdk/pull/13133
the rollback cmd need this to really work.

Reply:
the rollback was primarily cherry-picked because there were some conflicts for the IAVL upgrade commit

Reply:
I tried to cherry-pick those commits from the rollback fix PR -- I also had to cherry-pick https://github.com/cosmos/cosmos-sdk/commit/3bdbaf17912f2632d4700f6b471b5a3712bd00a9 because otherwise `Application` interface won't have `CommitMultiStore()`
https://github.com/cosmos/cosmos-sdk/compare/v0.44.6...crypto-org-chain:cosmos-sdk:v0.44.8-patch3
https://github.com/cosmos/cosmos-sdk/compare/release/v0.44.x...crypto-org-chain:cosmos-sdk:v0.44.8-patch3